BMM.Core/ViewModels/Interfaces/IBaseViewModel.cs

```diff
@@ -0,0 +1,11 @@
+using MvvmCross.Commands;
+using MvvmCross.ViewModels;
+
+namespace BMM.Core.ViewModels.Interfaces
+{
+    public interface IBaseViewModel : IMvxNotifyPropertyChanged
```

Comment:
I think this is dangerous. Calling it BaseViewModel implies that every viewModel should implement it.
But IsLoading is only relevant for ViewModels that load something from the server.
I think I would prefer a more specific name

Reply:
It's an improvemnt from before. So you can leave it